src/components/DepositWidget/Styled.tsx

```diff
@@ -85,12 +85,18 @@ export const TokenRow = styled.tr`
     padding: 0;
 
     li {
-      white-space: nowrap;
+      display: flex;
+      align-items: baseline;
+      justify-content: flex-end;
+
+      > span {
+        margin-right: 0.5rem;
+      }
     }
 
     button.wrapUnwrapEther {
       font-size: 1rem;
-      padding 0.2rem 0.5rem;
+      padding: 0.2rem 0.5rem;
```

Comment:
Heck is this?! Who left it there for 8 months?! Who do we `git blame` :point_right::man: ?
I almost want to [integrate stylelint](https://styled-components.com/docs/tooling#stylelint) or something, almost

Reply:
ANXO